_posts/2018-11-13-cs-enrollment.md

```diff
@@ -0,0 +1,105 @@
+---
+    title: Breaking Down the Computer Science Department -- Bit by Bit
+    teaser: As the tech world has been intensely growing, explore the changes in UCLA's CS department over time.
+    authors:
+    - keith_atienza
+    - lik_teng_ung
+    - mattie_sanseverino
+    key_takeaways: 
+    - Applications and enrollment in the CS department has greatly increased over the years.
+    - The student to faculty ratio within HSSEAS and the CS department has grown since 2013.
+    - Since 2009, upper and lower division lectures have increased in size. 
+    featured_image:
+    url: research-funding/graph.png
+    og_image: research-funding/graph.png
+    stylesheets: 
+    - /css/posts/cs-course-enrollment/tiles.css
+    scripts: 
+    - /js/posts/cs-enrollment/ratio-linechart.js
+    - /js/posts/cs-enrollment/bar_graph.js
+    - //cdn.plot.ly/plotly-latest.min.js
+    - //cdn.plot.ly/plotly-latest.min.js
+    - //d3js.org/d3.v5.min.js
+    - //use.typekit.com/dkx5efl.js
+    - //unpkg.com/d3-simple-slider
+    - /js/posts/cs-enrollment/tiles.js
+    - /js/posts/cs-enrollment/jquery-3.3.1.min.js
+    - /js/posts/cs-enrollment/waffle.js
+---
+
+<br> 
+## CS Applications in context
+
+<div class='waffle'></div>
+
+
+
+## CS vs. Engineering Faculty 
+<br>
+<p>Below you can see how the amount of faculty within the Computer Science department has changed relative to the faculty in the overall School of Engineering. </p>
+<br> 
+
+<iframe align="center" width="85%" height="400" frameborder="0" scrolling="no" src="//plot.ly/~MattieSanseverino/5.embed?link=false&modebar=false&width=100%&height=50%"></iframe>
```

Comment:
what's going on here? we shouldn't be using iframes. I thought plot.ly was a library, similar to d3?

Reply:
plot.ly doesn't have a good way to size the graph correctly and it kept turning up too large -- it was recommended to use iframe to fix this problem